src/client/Microsoft.Identity.Client/PublicApi/net462/PublicAPI.Shipped.txt

```diff
@@ -201,12 +201,13 @@ Microsoft.Identity.Client.AbstractApplicationBuilder<T>.WithLegacyCacheCompatibi
 Microsoft.Identity.Client.AbstractApplicationBuilder<T>.WithRedirectUri(string redirectUri) -> T
 Microsoft.Identity.Client.AbstractApplicationBuilder<T>.WithTelemetry(Microsoft.Identity.Client.ITelemetryConfig telemetryConfig) -> T
 Microsoft.Identity.Client.AbstractApplicationBuilder<T>.WithTenantId(string tenantId) -> T
+Microsoft.Identity.Client.AbstractConfidentialClientAcquireTokenParameterBuilder<T>.WithSignedHttpRequestProofOfPossession(Microsoft.Identity.Client.AppConfig.PoPAuthenticationConfiguration popAuthenticationConfiguration) -> T
```

Comment:
Sorry, I think I was not clear. There shouldn't be any changes to the "PublicAPI.Shipped.txt" files. Only to "PublicAPI.Unshipped.txt" files.